chain-api/src/utils/serialize.spec.ts

```diff
@@ -122,3 +123,10 @@ it("should handle very large numbers with no decimals", () => {
   expect(chainObjectS).toEqual(expectedClassS);
   expect(plainS).toEqual(expectedPlainS);
 });
+
+it("Bignumber toUtf8Bytes", () => {
+  const bigNumber = BigNumber("300");
+  console.log(`Type: ${typeof bigNumber}`);
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  expect(toUtf8Bytes(bigNumber as any)).toEqual(toUtf8Bytes("300"));
```

Comment:
I believe the type casting is no longer needed sice `toUtf8Bytes` accepts `string | BigNumber`

Reply:
Yep, that was old. Stripping it out!